frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -28,10 +28,16 @@ import { groupProjects } from './group-projects';
 import { ProjectGroup } from './ProjectGroup';
 
 const StyledApiError = styled(ApiError)(({ theme }) => ({
-    maxWidth: '400px',
+    maxWidth: '500px',
     marginBottom: theme.spacing(2),
 }));
 
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(4),
+}));
```

Comment:
So you moved the spacing here, right: why? For readability? I agree that it makes sense to let the parent control how items within it are spaced, but it adds another layer of nesting 🤷🏼